frontend/online-shop/src/styles/tableTheme.tsx

```diff
@@ -0,0 +1,29 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+
+const theme = createMuiTheme({
+  typography: {
+    fontFamily: ["sans-serif", "BlinkMacSystemFont"].join(","),
+    fontWeightBold: 1000,
+    fontSize: 22,
+  },
+  palette: {
+    primary: {
+      main: "#009879",
+    },
+    secondary: {
+      //tbody tr background color
```

Comment:
Remove such comments. Leave only the comments that are documenting complicate functions or business logic

Reply:
Fixed